src/Tizen.NUI/src/public/BaseComponents/VectorGraphics/Drawable.cs

```diff
@@ -52,6 +52,35 @@ public float Opacity
             }
         }
 
+        /// <summary>
+        /// The bounding box of the drawable object before any transformation.
+        /// @note The bounding box doesn't indicate the rendering region in the result but primitive region of the object.
+        /// @note The float type Rectangle class is not ready yet.
+        ///       Therefore, it transmits data in Vector4 class.
+        ///       This type should later be changed to the appropriate data type.
```

Comment:
you may use <remarks>.
reference: https://docs.microsoft.com/ko-kr/dotnet/csharp/programming-guide/xmldoc/recommended-tags-for-documentation-comments
```suggestion
        ///<remarks>The bounding box doesn't indicate the rendering region in the result but primitive region of the object.</remarks>
        ///<remarks> 
        ///       The float type Rectangle class is not ready yet.
        ///       Therefore, it transmits data in Vector4 class.
        ///       This type should later be changed to the appropriate data type.
        ///</remarks>
```

Reply:
I changed to use `<remarks>`